src/proto/primitive.proto

```diff
@@ -101,17 +101,17 @@ message PowerControl
 
 message Primitive
 {
+    reserved 5;
     oneof primitive
     {
         MovePrimitive move                    = 1;
         StopPrimitive stop                    = 2;
         DirectControlPrimitive direct_control = 3;
     }
+    Angle orientation = 4;
 
     // Sequence number for the primitive
-    uint64 sequence_number = 4;
-
-    reserved 5;
+    uint64 sequence_number = 7;
```

Comment:
don't change existing proto ids. keep `sequence_number` as 4 and orientation as `7`